.github/workflows/test.yml

```diff
@@ -0,0 +1,93 @@
+name: Test Application
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  # Print variables for logging and debugging purposes
+  checkEnv:
+    name: Check Env variables
+    runs-on: ubuntu-latest
+    steps:
+      - name: Print Env Vars
+        run: |
+          echo Git Base Ref: ${{ github.base_ref }}
+          echo Git Build ID: ${{ github.event.number }}
+          echo Git Pull Request Ref: ${{ github.event.pull_request.head.sha }}
+  test:
+    name: Test API in Docker env
+    runs-on: ubuntu-latest
+    if: ${{ github.event.pull_request.merged == false }}
+    env:
+      PROJECT_NAME: aest-sims
+      BUILD_ID: ${{ github.event.number }}
+      POSTGRES_PASSWORD: bcgov!
```

Comment:
Even that this password is used only for the test DB, consider moving to a secret.

Reply:
Same info is already exposed in sources/make file as default value .....